homeassistant/components/blink/camera.py

```diff
@@ -43,14 +43,14 @@ class BlinkCamera(Camera):
     """An implementation of a Blink Camera."""
 
     _attr_has_entity_name = True
-    _attr_name = None
```

Comment:
Same story as with the alarm panel. I'm guessing the name from the blink API is from blink, so before the name of the entity is the same as the device.